lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -446,23 +446,19 @@ sub annotate_process {
 		}
 	}
 	if (defined($parameters->{call_features_CDS_prodigal}) && $parameters->{call_features_CDS_prodigal} == 1)	{
-		if ($tax_domain ne 'U' ) {
-			if (@{$inputgenome->{features}} > 0) {
-#				$inputgenome->{features} = [];
-				$message .= "The existing gene features were cleared due to selection of gene calling with Glimmer3 or Prodigal.\n";
-			}
-			if (length($genecalls) == 0) {
-				$genecalls = "Standard gene features were called using: ";
-			} else {
-				$genecalls .= "; ";
-			}
-			$genecalls .= "prodigal";
-			push(@{$workflow->{stages}},{name => "call_features_CDS_prodigal"});
-			if (!defined($contigobj)) {
-				Bio::KBase::utilities::error("Cannot call genes on genome with no contigs!\n");
-			}
+		if (@{$inputgenome->{features}} > 0) {
+#			$inputgenome->{features} = [];
+			$message .= "The existing gene features were cleared due to selection of gene calling with Glimmer3 or Prodigal.\n";
+		}
+		if (length($genecalls) == 0) {
+			$genecalls = "Standard gene features were called using: ";
 		} else {
-			$message .= "Did not predict prodigal genes because the domain is $parameters->{domain}\n\n";	
+			$genecalls .= "; ";
+		}
+		$genecalls .= "prodigal";
```

Comment:
This doesn't make too much sense to me. I assume it $genecalls becomes part of the report? Why append "; prodigal" if no genes get called?

Reply:
This section of code is describing what the user selected. $genecalls is part of a longer message that is part of the messaging at the end. It is a list of all the tools that the user requested. The +/- statements in the commit make it hard to follow. I'm just putting these statements back they way there were a couple of commits ago and they parallel the glimmer section (lines 427-443). 

If the user selected prodigal but the $genecalls is still blank, prepend with "Standard gene features were called using: ";

Reply:
👍